sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 * Fixed the fail back flow where not all partitions were failing back to original first preferred region for Per-Partition Circuit Breaker. - [PR 44099](https://github.com/Azure/azure-sdk-for-java/pull/44099)
 * Fixed diagnostics issue where operations in Gateway mode hitting end-to-end timeout would not capture diagnostics correctly. - [PR 44099](https://github.com/Azure/azure-sdk-for-java/pull/44099)
 * Enabled `excludeRegions` to be applied for `QueryPlan` calls. - [PR 45196](https://github.com/Azure/azure-sdk-for-java/pull/45196)
+* Enabled Per-Partition Circuit Breaker to kick in for requests part of availability strategy. - [PR 45244](https://github.com/Azure/azure-sdk-for-java/pull/45244)
```

Comment:
I don't really get what the summary is trying to say - would be good to explain a bit what "requests part" is meaning

Reply:
Ugh - seems to be a "mis"-push. Will fix it.

Reply:
Fixed